support/network_of_terms_client.erl

```diff
@@ -43,8 +43,14 @@ lookup(Uris, Context) ->
 %% @doc Get list of term sources that are available in the Network of Terms.
 -spec get_sources() -> list(map()).
 get_sources() ->
-    #{<<"data">> := #{<<"sources">> := Sources}} = request(sources_query()),
-    Sources.
+    case request(sources_query()) of
+        #{<<"data">> := #{<<"sources">> := Sources}} ->
+            Sources;
+        Other ->
+            log_unexpected_data(sources_query, Other),
+            []
+    end.
```

Comment:
I wonder if we would like to factor out the pattern match, so we could write the queries as `get_sources() -> request(<<"sources">>, sources_query()).` etc.

Reply:
On second thought, let's not, because of the limited gain.